### PR TITLE
Inject new literals at the end of file, rather than beginning

### LINKED ISSUE #288: Improve Style of Code Generated by DefineConstantForLiteral Codemod
The `DefineConstantForLiteral` generates code with two style issues:

* the constant names have no `_` characters to delineate words
* the constants are not indented within the enclosing class as one would expect.

Here's an example of a constant added by this codemod:

```java
class Foo {
private static final String APICERTIFICATE = "APICertificate";
```

Expected behavior:

```java
class Foo {
  private static final String API_CERTIFICATE = "APICertificate";
```

### PR DESCRIPTION
**Pull request by nahsra**
Fixes #288

### CODE REVIEW

**Review by gilday — APPROVED** [review]
lgtm

File: core-codemods/src/main/java/io/codemodder/codemods/CreateConstantForLiteral.java [hunk 1]
```diff
@@ -72,14 +72,14 @@ protected void defineConstant(final String constantName) {
...
+   * href="https://github.com/pixee/codemodder-java/issues/288">preferred by users and better for
+   * JavaParser to match the existing indentation</a>.
    */
-  protected void addConstantFieldToClass(final FieldDeclaration constantField) {
-
+  private void addConstantFieldToClass(final FieldDeclaration constantField) {
     final NodeList<BodyDeclaration<?>> members = classOrInterfaceDeclaration.getMembers();
-
-    members.addFirst(constantField);
+    members.addLast(constantField);
```
Comment (ryandens) [inline core-codemods/src/main/java/io/codemodder/codemods/CreateConstantForLiteral.java:82]: Out of curiosity, how does this change impact the indentation in some of the test files?

**Review by ryandens — APPROVED** [review]

File: core-codemods/src/main/java/io/codemodder/codemods/CreateConstantForLiteral.java (hunk 1 above)
Comment (nahsra) [reply core-codemods/src/main/java/io/codemodder/codemods/CreateConstantForLiteral.java:82]: See the discussion in #288

Comment (ryandens) [reply core-codemods/src/main/java/io/codemodder/codemods/CreateConstantForLiteral.java:82]: Nice!

**Pull request merged**